source/opt/debug_info_manager.cpp

```diff
@@ -441,32 +440,37 @@ bool DebugInfoManager::IsAncestorOfScope(uint32_t scope, uint32_t ancestor) {
   return false;
 }
 
-Instruction* DebugInfoManager::GetDebugLocalVariableFromDeclare(
-    Instruction* dbg_declare) {
-  assert(dbg_declare);
+bool DebugInfoManager::IsDeclareVisibleToInstr(Instruction* dbg_declare,
+                                               Instruction* scope) {
+  assert(dbg_declare != nullptr && scope != nullptr);
```

Comment:
nit: can you split this into two asserts? this way you can tell which pointer was at fault without running under a debugger

Reply:
done